File: examples/fibonacci.cpp
#include <iostream>
using namespace std;

int main() {
    int number = 10, n1 = 0, n2 = 1, nextTerm;

    cout << "Fibonacci Series in C++:" << endl;

    for (int i = 1; i <= number; ++i) {
        cout << n1 << endl;
        nextTerm = n1 + n2;
        n1 = n2;
        n2 = nextTerm;
    }

    return 0;
}

File: examples/Fibonacci.java
public class Fibonacci {
    public static void main(String[] args) {
        int number = 10, n1 = 0, n2 = 1;
        System.out.println("Fibonacci Series in Java:");

        for (int i = 1; i <= number; ++i) {
            System.out.println(n1);
            int nextTerm = n1 + n2;
            n1 = n2;
            n2 = nextTerm;
        }
    }
}